test/integration/runner_task_test.rb

```diff
@@ -18,7 +18,7 @@ def test_run_without_verify_result_succeeds_as_soon_as_pod_is_successfully_creat
       "Result: SUCCESS",
       "Result verification is disabled for this task",
       "The following status was observed immediately after pod creation:",
-      %r{Pod/task-runner-\w+\s+Pending},
+      %r{Pod/task-runner-\w+\s+(Pending|Running)},
```

Comment:
This change and the one below are to decrease test flakiness. It's not important to these two test cases whether the pod manages to start running in the run window.